src/components/BRB/BRBMainComponent.tsx

```diff
@@ -535,6 +541,7 @@ const BrbWrapper = styled(ItemV)`
   justify-content: center;
   align-items: center;
   background: #000;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family